src/Workspaces/Remote/Core/SolutionAssetStorage.cs

```diff
@@ -115,18 +116,9 @@ internal TestAccessor(SolutionAssetStorage solutionAssetStorage)
             _solutionAssetStorage = solutionAssetStorage;
         }
 
-        public async ValueTask<SolutionAsset?> GetAssetAsync(Checksum checksum, CancellationToken cancellationToken)
+        public async ValueTask<SolutionAsset> GetRequiredAssetAsync(Checksum checksum, CancellationToken cancellationToken)
```

Comment:
this is inside a test accessor.  and all callers were throwing if we got null anyways.